mlflow/server/auth/__init__.py

```diff
@@ -381,7 +382,8 @@ def get_before_request_handler(request_class):
 
 
 @catch_mlflow_exception
-def _before_request():
+def _before_request(request):
+    return
```

Comment:
see https://www.starlette.io/authentication/ for basic auth middleware. It requires password verification inside the middleware, so might need to change the code design a little bit.